styles/styles.css

```
@@ -223,24 +224,24 @@
   --loading-background: #ededed;
 
   /* fonts */
-  --ff-gilroy-xbold: 'Gilroy ExtraBold 700', 'Gilroy ExtraBold 700 Fallback' , sans-serif;
+  --ff-gilroy-xbold: 'Gilroy ExtraBold 700', 'Gilroy ExtraBold 700 Fallback', sans-serif;
   --ff-gilroy-xbold-italic: 'Gilroy ExtraBold Italic 700', 'Gilroy ExtraBold Italic 700 Fallback', sans-serif;
   --ff-gilroy-bold: 'Gilroy Bold 700', 'Gilroy Bold 700 Fallback', sans-serif;
   --ff-gilroy-semibold: 'Gilroy SemiBold 600', 'Gilroy SemiBold 600 Fallback', sans-serif;
   --ff-gilroy-medium: 'Gilroy Medium 500', 'Gilroy Medium 500 Fallback', sans-serif;
   --ff-gilroy-regular: 'Gilroy Regular 400', 'Gilroy Regular 400 Fallback', sans-serif;
   --ff-gilroy-light: 'Gilroy Light 300', 'Gilroy Light 300 Fallback', sans-serif;
   --ff-gilroy-ultralight: 'Gilroy UltraLight 200', 'Gilroy UltraLight 200 Fallback', sans-serif;
-  --ff-zahrah-bold: 'Zahrah Bold 700', 'Zahrah Bold 700 Fallback' , serif;
-  --ff-zahrah-bold-italic: 'Zahrah Bold Italic 700', 'Zahrah Bold Italic 700 Fallback', serif;
-  --ff-zahrah-semibold: 'Zahrah Semibold 600', 'Zahrah Semibold 600 Fallback', serif;
-  --ff-zahrah-semibold-italic: 'Zahrah Semibold Italic 600', 'Zahrah Semibold Italic 600 Fallback', serif;
-  --ff-zahrah-medium: 'Zahrah Medium 500', 'Zahrah Medium 500 Fallback', serif;
-  --ff-zahrah-medium-italic: 'Zahrah Medium Italic 500', 'Zahrah Medium Italic 500 Fallback', serif;
-  --ff-zahrah-regular: 'Zahrah Regular 400', 'Zahrah Regular 400 Fallback', serif;
-  --ff-zahrah-regular-italic: 'Zahrah Regular Italic 400', 'Zahrah Regular Italic 400 Fallback', serif;
-  --ff-zahrah-light-italic: 'Zahrah Light Italic 300', 'Zahrah Light Italic 300 Fallback', serif;
-  --ff-zahrah-light: 'Zahrah Light 300', 'Zahrah Light 300 Fallback', serif;
+  --ff-zahrah-bold: 'Zahrah Bold', serif;
```

```suggestion
  --ff-zahrah-bold: 'Zahrah Bold', 'Zahrah Bold Fallback', serif;
```
You forgot about all fallback fonts in the Zahrah font variables.